recipes/up42-py/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "up42-py" %}
+{% set version = "0.20.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/up42-py-{{ version }}.tar.gz
+  sha256: 4a94622cdbca24a0d2adc1a67a94f1f426f7b46d632540a36b680b50cf3a2b85
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7,<3.10
+  run:
+    - branca
+    - click
+    - descartes
+    - folium
+    - geojson
+    - geopandas
+    - matplotlib-base
+    - python >=3.7,<3.10
+    - rasterio
+    - requests
+    - requests-oauthlib
+    - tenacity
+    - tqdm
+
+test:
+  imports:
+    - up42
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/up42/up42-py
+  summary: Python SDK for UP42
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://sdk.up42.com
+  dev_url: https://github.com/up42/up42-py
+
+extra:
+  recipe-maintainers:
+    - up42
```

Comment:
I believe we don't allow organizations listed, just actual github users. Also please confirm that the other co-organizer has agreed to be listed.